File: laba3/lab3.swift
import Foundation

func searchEven(_ num: Int) -> Int {
    var num = num
    var count = 0
    while num > 0 {
        let last = num % 10
        if last % 2 == 0 {
            count += 1
        }
        num /= 10
    }
    return count
}

print("Enter number of numbers: ", terminator: "")
if let nStr = readLine(), let n = Int(nStr) {
    for _ in 0..<n {
        print("Enter number: ", terminator: "")
        if let kStr = readLine(), let k = Int(kStr) {
            print(searchEven(k))
        }
    }
}

File: laba3/lab3.go
package main

import (
    "fmt"
)

func SearchEven(num int) int {
    count := 0
    for num > 0 {
        last := num % 10
        if last%2 == 0 {
            count++
        }
        num /= 10
    }
    return count
}

func main() {
    var n int
    fmt.Print("Enter number of numbers: ")
    fmt.Scan(&n)
    for i := 0; i < n; i++ {
        var k int
        fmt.Print("Enter number: ")
        fmt.Scan(&k)
        fmt.Println(SearchEven(k))
    }
}
	